tests/fixtures/issues/issue-436.php

```diff
@@ -1538,59 +1538,6 @@
         'lite' => false,
         'standard' => true,
     ],
-    'issue-436-AE' => [
-        'ua' => 'Mozilla/5.0 (iPad; CPU iPhone OS 501 like Mac OS X) AppleWebKit/534.46 (KHTML like Gecko) Version/5.1 Mobile/9A405 Safari/7534.48.3',
```

Comment:
I removed this one from the suite due to the iOS version number (`504`). We do see versions like this, but I've never seen it from an actual device that I can verify, so assuming these are fake.